react/react1/week2/todo-list/src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import Todo from "./components/Todo"
+import Header from './components/Header';
+import Footer from './components/Footer';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+   <Header />
+    <Todo />
+    <Footer />
```

Comment:
It's a more common practice to keep a main component here rather than having multiple of them. So it would be a better idea to have a separate file which includes these and put that component here, which usually is `App`

Reply:
yaah. you are right. I will update code.